bot/exts/info/information.py

```diff
@@ -524,13 +524,16 @@ async def json(self, ctx: Context, message: Message) -> None:
         await self.send_raw_content(ctx, message, json=True)
 
     @command(aliases=("rule",))
-    async def rules(self, ctx: Context, *args: Optional[str]) -> Optional[Set[int]]:
+    async def rules(self, ctx: Context, *, args: Optional[str]) -> Optional[Set[int]]:
         """
         Provides a link to all rules or, if specified, displays specific rule(s).
 
         It accepts either rule numbers or particular keywords that map to a particular rule.
         Rule numbers and keywords can be sent in any order.
         """
+        # Note: *args cannot be used due to a discord.py bug
+        # causing infinite loops during greedy string parsing.
+
```

Comment:
I don't think this comment is needed, as *args in a command shouldn't be a thing that's used, and having a comment here doesn't stop someone adding it to a new command, that can only be done in PRs.